app/lib/url.rb

```diff
@@ -46,7 +46,7 @@ def self.reaction(reaction)
   #
   # @param tag [Tag] the tag to create the URL for
   def self.tag(tag, page = 1)
-    url(["/t/#{tag.name}", ("/page/#{page}" if page > 1)].join)
+    url(["/t/#{CGI.escape(tag.name)}", ("/page/#{page}" if page > 1)].join)
```

Comment:
I was curious about this, too.

Reply:
IIRC we had setup `URL` because of the indirection of the `StoriesController` and the lack of namespacing, but `tag` is namespaced, so that should work correctly